test/125_observers_cant_vote_3_test.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash
+
+. ./config.sh
+
+start_suite "Observers can't vote"
+
+# Start network that doesn't have enough peers for consensus
+weave_on $HOST1 launch --init-peer-count=2
+weave_on $HOST2 launch --init-peer-count=0 $HOST1
+
+# Check that an allocation times out
+assert_raises "timeout 10 cat <( start_container $HOST1 )" 124
```

Comment:
Not true. That is for cases where we don't expect to hit the timeout. Note that U've not complained about any of those.